Code/GraphMol/Chirality.cpp

```diff
@@ -32,6 +32,333 @@ bool shouldDetectDoubleBondStereo(const Bond *bond) {
   return (!ri->numBondRings(bond->getIdx()) ||
           ri->minBondRingSize(bond->getIdx()) > 7);
 }
+
```

Comment:
for anyone else reviewing this PR:
This code (and the function below) was just moved from `MolFileStereochem.cpp`.
It may be worth reviewing/refactoring it, but that's probably not in scope for this PR.